zebra-chain/src/transaction/tests/vectors.rs

```diff
@@ -315,3 +315,29 @@ fn fake_v5_round_trip_for_network(network: Network) {
         );
     }
 }
+
+#[test]
+fn invalid_orchard_nullifier() {
```

Comment:
@dconnolly would it also be useful to proptest on random bytes here?
Or are we essentially duplicating the logic inside `try_from`?

Reply:
> @dconnolly would it also be useful to proptest on random bytes here?
> Or are we essentially duplicating the logic inside `try_from`?

It kinda reproduces that yeah, I'm not sure if there's a strong testing value there